src/js/components/SiteHeader.tsx

```diff
@@ -45,7 +44,7 @@ const SiteHeader = () => {
           <Space>
             {translated && (
               <Button shape="round" onClick={changeLanguage}>
-                {language}
+                {i18n.language === 'en' ? 'Français' : 'English'}
```

Comment:
look up language full name in translation or in a dictionary instead of hard-coding, but i like this change overall